src/Lorentz/Contracts/RegistryDAO.hs

```diff
@@ -0,0 +1,159 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+-- | A Registry DAO can be used to store key value pair @(k, v)@
+module Lorentz.Contracts.RegistryDAO
+  ( registryDaoContract
+  , config
+  )
+  where
+
+import Lorentz
+
+import qualified Lorentz.Contracts.BaseDAO as DAO
+import Lorentz.Contracts.RegistryDAO.Types
+import qualified Lorentz.Contracts.BaseDAO.Types as DAO
+import Universum hiding (drop, swap, (>>), some)
+import Util.Markdown
+
+-- | Proposal check: `s = size(pack(proposalMetadata)) && s < s_max` `(a * s + b)`
```

Comment:
Analog of markdown's ` is @, so please use it instead. You can do `make haddock-no-deps` to check rendered documentation.

Reply:
I've improved the haddock comments for most functions now.